doc/Manual.md

```diff
@@ -211,76 +212,84 @@ stopping and restarting it while trying different port configurations.
+a `logcfg.dat` file in that directory overrides the default. If you are using
+SSB only in a contest you can force TLF to start in SSB mode by uncommenting
+the SSBMODE keyword in `logcfg.dat`. 
 
 If you have adopted the method that loops back the mic audio and you want to
 automatically mute the mic when voice keyer messages are played, copy the
-'play_vk' shell script found in the 'scripts' directory of your TLF release to
+`play_vk` shell script found in the `scripts` directory of your TLF release to
 the same directory. This will then be used in preference to the default.
-Uncomment the lines beginning with 'amixer'. 
-
-Since tlf-1.1.0, (un)muting and playing voice messages has been devolved to
-this external script file, because not all soundcards offer the same
-interface. 'Mic' could be called something else. One way of finding this out
-is to run 'amixer' from a terminal which returns its capabilities. An
-alternative method is to install your distro's version of the 'alsamixergui'
-package and simply see what's available on the faders. This is probably a good
-idea anyway because it's likely sound won't work without some manual
-intervention. 
-
-If you have something other than a 'Mic' source you could try replacing 'Mic'
-with what you think it should be in the local 'play_vk' file. The process of
-muting and unmuting the mic is displayed in alsamixergui, which is useful for
-debug purposes. Please share any information you discover with the TLF
-community. 
-
-Similarly, a 'rules' sub-directory can contain a contest specific rule file.
-This rule file must contain the paths to the audio files corresponding to each
-F-key to be used. An example might be: 
+Uncomment the lines beginning with `amixer`. 
+
+Since TLF version 1.1.0, (un)muting and playing voice messages has been
+devolved to this external script file, because not all soundcards offer the
+same interface. 'Mic' could be called something else. One way of finding this
+out is to run `amixer` from a terminal, which returns its capabilities. An
+alternative method is to install your distribution's version of the
+`alsamixergui` package and simply see what's available on the faders. This is
+probably a good idea anyway because it's likely sound won't work well without
+some manual intervention. 
+
+If you have something other than a 'Mic' source, you could try replacing 'Mic'
+with what you think it should be in the local `play_vk` file. The process of
+muting and unmuting the mic is displayed in `alsamixergui`, which is useful
+for debug purposes. Please share any information you discover with the TLF
+community.
+
+A `rules` sub-directory within the local contest folder should contain a
+contest-specific rule file. As with `logcfg.dat`, this local copy will
+override the defaults, which you will want to do. This rule file must contain
+the paths to the audio files corresponding to each F-key to be used. An
+example might be: 
 
 ```
```

Comment:
line 312: "f12" -> "F12"

"S+P" is referred to as "S&P" in other parts of the manual and the man page.

Reply:
Fixed.